### ISSUE TITLE
No interface is running right now. is it a gradio outage problem today? May 11, 2023, Thursday 11:45 AM?

### ISSUE THREAD

**Issue by ganeshrk1342** [TheLastBen/fast-stable-diffusion#2155]
No interface is running right now. is it a gradio outage problem today? May 11, 2023, Thursday 11:45 AM?

_Originally posted by @ganeshrk1342 in https://github.com/TheLastBen/fast-stable-diffusion/issues/1489#issuecomment-1544281270_

**Comment by ganeshrk1342** [comment]
![Start_Stable-Diffusion](https://github.com/TheLastBen/fast-stable-diffusion/assets/132179914/ce0b4606-8180-4c91-b3fe-0adb030c2681)

I got the same error 'No interface is running right now' even though GColab is connected.

**Comment by TheLastBen** [comment]
use ngrok

**Comment by ganeshrk1342** [comment]
Hi Ben,
How do I use it? Not familiar with this. Thanks

**Comment by TheLastBen** [comment]
get your token here https://dashboard.ngrok.com/get-started/your-authtoken and insert it in the last cell

**Comment by ganeshrk1342** [comment]
Thank You Ben. But I am still having trouble at the last step. 

from pyngrok import ngrok, conf
      7 
      8 Use_Cloudflare_Tunnel = False #@param {type:"boolean"}

ModuleNotFoundError: No module named 'pyngrok'

How do I install pyngrok module? I did install at the beginning using code but did not work. For the ngrok token i even filled out username and the password. Do I have to? os is it optional?

**Comment by TheLastBen** [comment]
pyngrok is automatically installed by the requirements cell, you are probably disconnected from the runtime because either you're using free colab or you missed the previous cells

**Comment by ganeshrk1342** [comment]
Thanks Ben. That worked. But now this ...

Model loaded in 40.3s (calculate hash: 16.7s, load weights from disk: 13.1s, create model: 6.4s, apply weights to model: 2.5s, apply half(): 0.9s, move model to device: 0.7s).
Running on local URL: https://af23-35-204-77-132.ngrok-free.app/
✔ Connected

I upgraded to google colab pro and used runtime GPU T4 and got connected. I clicked on the link and that took me to a login prompt. My username and password for ngrok doesn't work. What is my username and password?
